Library/Formula/mpw.rb

```diff
@@ -0,0 +1,32 @@
+require "formula"
+
+class Mpw < Formula
+  homepage "http://masterpasswordapp.com"
+  url "https://ssl.masterpasswordapp.com/mpw-2.1-cli3-3-gd5ff215.tar.gz"
```

Comment:
The release is 2.1-cli3.  3 points away due some homebrew-specific build script changes.